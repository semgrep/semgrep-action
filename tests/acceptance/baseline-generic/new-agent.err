=== Running: SEMGREP_BASELINE_COMMIT="HEAD~1" SEMGREP_RULES="r/python.lang.correctness.useless-eqeq.useless-eqeq" semgrep ci --no-suppress-errors
Running `semgrep ci` without API token but with configs ('r/python.lang.correctness.useless-eqeq.useless-eqeq',)


╭────────────────╮
│ Debugging Info │
╰────────────────╯

  SCAN ENVIRONMENT
  versions    - semgrep x.y.z on python x.y.z
  environment - running in environment git, triggering event is unknown


╭─────────────╮
│ Scan Status │
╰─────────────╯

Fetching configuration from semgrep.dev
Semgrep rule registry URL is https://semgrep.dev/registry.
Scanning 1 file.
  Current version has 1 finding.

Skipping baseline scan, because all current findings are in files that didn't exist in the baseline commit.


╭─────────╮
│ Results │
╰─────────╯



╭──────────────╮
│ Scan Summary │
╰──────────────╯

Some files were skipped or only partially analyzed.
  Scan was limited to files changed since baseline commit.

CI scan completed successfully.
  Found 1 finding (1 blocking) from 1 rule.
  Has findings for blocking rules so exiting with code 1
