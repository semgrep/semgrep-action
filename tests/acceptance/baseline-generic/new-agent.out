

┌─────────────────────────┐
│ 1 Blocking Code Finding │
└─────────────────────────┘

    bar.py
       python.lang.correctness.useless-eqeq.useless-eqeq
          This expression is always True: `2 == 2` or `2 != 2`. If testing for
  floating point NaN, use
          `math.isnan(2)`, or `cmath.isnan(2)` if the number is complex.
          Details: https://sg.run/GeAp

            1┆ 2 == 2

  BLOCKING CODE RULES FIRED:
    python.lang.correctness.useless-eqeq.useless-eqeq

